core/xmp_parser.cpp

```diff
@@ -0,0 +1,137 @@
+#include "xmp_parser.h"
+#include "subsurface-string.h"
+#include "dive.h" // TODO: monster-include file only for utc_mktime.
```

Comment:
hrmpf - just copy the one element that you need? that seems overkill.

Reply:
Done.